### PR TITLE
WINDUP-754 Group the identical hint labels with number -

### PR DESCRIPTION
**Pull request by OndraZizka**
e.g.: Oracle proprietary type reference 5×

### CODE REVIEW

File: rules-java/api/src/main/resources/reports/templates/java_application.ftl [hunk 1]
```diff
@@ -64,11 +64,18 @@
...
+                    <#assign map = {}>
                     <#list sourceReportModel.sourceFileModel.classificationModels.iterator() as classification>
-                        <li class='warning clsf'>${classification.classification?html}</li>
+                        <#assign count = (map[classification.classification]!0) + 1>
+                        <#assign map += {classification.classification : count}>
                     </#list>
                     <#list sourceReportModel.sourceFileModel.inlineHints.iterator() as hintLine>
-                        <li class='warning hint'>${hintLine.title?html}</li>
+                        <#assign count = (map[hintLine.title]!0) - 1>
+                        <#assign map += {hintLine.title : count}>
```
Comment (OndraZizka) [inline rules-java/api/src/main/resources/reports/templates/java_application.ftl]: If there are same hint lines as classification titles, this trick would not work well... Can redo if you think it's likely.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1202/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/574/

File: rules-java/api/src/main/resources/reports/templates/java_application.ftl (hunk 1 above)
Comment (jsight) [inline rules-java/api/src/main/resources/reports/templates/java_application.ftl]: I think that it is completely possible for classification to have the same title as a hint.

Comment (mareknovotny) [inline rules-java/api/src/main/resources/reports/templates/java_application.ftl]: the numbers (groupping of titles) look fine

Comment (mareknovotny) [inline rules-java/api/src/main/resources/reports/templates/java_application.ftl]: the screenshot taken for different issue, but it shows this PR result too https://issues.jboss.org/secure/attachment/12394886/Screenshot%20from%202015-11-12%2014-51-16.png

Comment (jsight) [inline rules-java/api/src/main/resources/reports/templates/java_application.ftl]: What happens if classification and hint have the same title?

Comment (OndraZizka) [inline rules-java/api/src/main/resources/reports/templates/java_application.ftl]: The count would be wrong. They would cancel each other's "1". I'll improve this

**Comment by OndraZizka** [comment]
Now the counting works fine.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1217/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/589/

**Pull request merged**